cg/apps/demultiplex/sample_sheet/create.py

```diff
@@ -14,9 +16,9 @@
 
 def get_sample_sheet_creator(
     flow_cell: FlowCellDirectoryData,
-    lims_samples: list[FlowCellSample],
+    lims_samples: list[FlowCellSampleBcl2Fastq | FlowCellSampleBCLConvert],
```

Comment:
I did this just to get rid of some pycharm warnings. This will be reduced to only one class when BCL2Fastq is deprecated. I will leave it as it is and later it would be clearer with the one class